src/ChunkDef.h

```diff
@@ -31,8 +31,9 @@ class cEntity;
 class cClientHandle;
 class cBlockEntity;
 
-typedef std::list<cEntity *>        cEntityList;
-typedef std::list<cBlockEntity *>   cBlockEntityList;
+typedef std::unique_ptr<cEntity> StoredEntity;
```

Comment:
I know it's nitpicky, but how about `OwnedEntity` instead? It makes more sense in my opinion :)